### ISSUE TITLE
text color remains black when a color parameter is specified

### ISSUE THREAD

**Issue by ngaurav** [dnfield/flutter_svg#278]
When color parameter is used to change color of the SVG, the color of text elements remain unaffected.

**Comment by dnfield** [comment]
Please share a reproduction case.

**Comment by ngaurav** [comment]
```xml
SvgPicture.string(
    '<svg class="bob" font-family="arial" font-size="14" height="160" width="88" xmlns="http://www.w3.org/2000/svg"><defs><marker id="triangle" markerHeight="8" markerWidth="8" orient="auto" refX="4" refY="2" viewBox="0 0 8 4"><polygon fill="black" points="0,0 0,4 8,2 0,0"/></marker><marker id="clear_triangle" markerHeight="10" markerWidth="10" orient="auto" refX="1" refY="7" viewBox="0 0 20 14"><polygon fill="none" points="2,2 2,12 18,7 2,2" stroke="black" stroke-width="2"/></marker><marker id="circle" markerHeight="5" markerWidth="5" orient="auto" refX="10" refY="10" viewBox="0 0 20 20"><circle cx="10" cy="10" fill="black" r="8"/></marker><marker id="square" markerHeight="5" markerWidth="5" orient="auto" refX="10" refY="10" viewBox="0 0 20 20"><rect fill="black" height="20" width="20" x="0" y="0"/></marker><marker id="open_circle" markerHeight="10" markerWidth="10" orient="auto" refX="10" refY="10" viewBox="0 0 20 20"><circle cx="10" cy="10" fill="white" r="4" stroke="black" stroke-width="2"/></marker><marker id="big_open_circle" markerHeight="20" markerWidth="20" orient="auto" refX="20" refY="20" viewBox="0 0 40 40"><circle cx="20" cy="20" fill="white" r="6" stroke="black" stroke-width="2"/></marker></defs><g stroke="#000" stroke-linecap="round" stroke-width="2" stroke-opacity="1" fill-opacity="1" stroke-linejoin="miter"><g><line x1="0" x2="28" y1="40" y2="40"/><line x1="28" x2="28" y1="16" y2="40"/><line x1="28" x2="28" y1="40" y2="72"/><line x1="28" x2="60" y1="40" y2="40"/><line x1="28" x2="28" y1="72" y2="104"/><line x1="28" x2="60" y1="72" y2="72"/><line x1="28" x2="28" y1="104" y2="136"/><line x1="28" x2="60" y1="104" y2="104"/><line x1="28" x2="28" y1="136" y2="160"/><line x1="28" x2="60" y1="136" y2="136"/><line x1="60" x2="60" y1="16" y2="40"/><line x1="60" x2="60" y1="40" y2="72"/><line x1="60" x2="88" y1="40" y2="40"/><line x1="60" x2="60" y1="72" y2="104"/><line x1="60" x2="88" y1="72" y2="72"/><line x1="60" x2="60" y1="104" y2="136"/><line x1="60" x2="88" y1="104" y2="104"/><line x1="60" x2="60" y1="136" y2="160"/><line x1="60" x2="88" y1="136" y2="136"/></g><g><line x1="0" x2="28" y1="72" y2="72"/></g><g><line x1="0" x2="28" y1="104" y2="104"/></g><g><line x1="0" x2="28" y1="136" y2="136"/></g><g><text stroke-width="1" x="9" y="28">2</text></g><g><text stroke-width="1" x="9" y="60">2</text></g><g><text stroke-width="1" x="9" y="92">2</text></g><g><text stroke-width="1" x="9" y="124">2</text></g><g><text stroke-width="1" x="9" y="156">2</text></g><g><text stroke-width="1" x="33" y="28">18</text></g><g><text stroke-width="1" x="41" y="60">9</text></g><g><text stroke-width="1" x="41" y="92">4</text></g><g><text stroke-width="1" x="41" y="124">2</text></g><g><text stroke-width="1" x="41" y="156">1</text></g><g><text stroke-width="1" x="73" y="28">0</text></g><g><text stroke-width="1" x="73" y="60">1</text></g><g><text stroke-width="1" x="73" y="92">0</text></g><g><text stroke-width="1" x="73" y="124">0</text></g><g><text stroke-width="1" x="73" y="156">1</text></g></g></svg>',
    color: Colors.white,
);
```
Here the text remains black. But the color filter gets successfully applied on lines.

**Comment by dnfield** [comment]
Fixed in v0.17.0
